app/src/test/java/ca/llamabagel/transpo/ui/trips/ClearStopCacheUseCaseTest.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019 Derek Ellis. Subject to the MIT license.
+ */
+
+package ca.llamabagel.transpo.ui.trips
+
+import ca.llamabagel.transpo.data.TestStops
+import ca.llamabagel.transpo.data.provideFakeTripsRepository
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.*
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class ClearStopCacheUseCaseTest {
+
+    private val repository = provideFakeTripsRepository()
+    private val clearStopCache = ClearStopCacheUseCase(repository)
+
+    @Test
+    fun `clear stop cache`() = runBlocking<Unit> {
```

Comment:
Again more descriptive. What is the test checking for. Tests names should usually be "when ___, then ___"